packages/api-gateway/src/drivers/http/adapters/users/handlers.js

```diff
@@ -42,12 +42,12 @@ async function createUser(req, reply) {
     });
 
     if (result.isBoom === true) {
-      return errorHandler(result, reply)
+      return errorHandler(result, reply);
     }
 
     return reply
       .code(200)
-      .header("Content-Type", "application/json; chartset:utf-8")
+      .header('Content-Type', 'application/json; chartset:utf-8')
```

Comment:
nit-pick: I think that you don't need to specify the content-type, fastify will set it for you automatically, can you remove this line?